core/domain/classifier_services.py

```diff
@@ -214,6 +214,13 @@ def save_classifier(classifier):
         classifier_id = _create_classifier(classifier)
     else:
         _update_classifier(classifier_data_model, classifier.state_name)
+
+    # Create the mapping from <exp_id,exp_version,state_name>
+    # to classifier_id.
+    create_classifier_exploration_mapping(
```

Comment:
I'm not really sure about when classifiers will be updated? @anmolshkl When might we need it? These functions were implemented before I got started with the Answer Classification Project.

@seanlip Even if there is a case where we update the classifier, it will most certainly happen due to the exploration/state being updated. So, we will definitely have a new <exp_id, exp_version, state_name> to map to the updated classifier. So, I think even if we do update the classifier, a new mapping has to be created.